roles/ceph-common/defaults/main.yml

```diff
@@ -174,9 +174,39 @@ debug_mds_level: 20
 rbd_cache: "true"
 rbd_cache_writethrough_until_flush: "true"
 rbd_concurrent_management_ops: 20
+
 rbd_client_directories: false # this will create rbd_client_log_path and rbd_client_admin_socket_path directories with proper permissions
```

Comment:
I thought we were going to keep it? In #687 you say:

"I think the right approach would be to disable the creation of the directories by default."

I've added the `when:` clause back in to directory creation. If you want me to remove it instead, I can do that.